internal/plugins/manifests/v2/init.go

```diff
@@ -15,45 +15,237 @@
 package v2
 
 import (
+	"errors"
 	"fmt"
+	"github.com/operator-framework/operator-sdk/internal/plugins/manifests/v2/internal/templates/config/samples"
```

Comment:
No need to have another internal level
```suggestion
	"github.com/operator-framework/operator-sdk/internal/plugins/manifests/v2/templates/config/samples"
```
Also fix import ordering